### PR TITLE
mamali/update Reset password design with the universal flow

### PR DESCRIPTION
**Pull request by reza-fs**
# Description

Summary

Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by AbdallaMourad — APPROVED** [review]

File: src/pages/reset-password/index.js [hunk 1]
```diff
@@ -70,82 +78,117 @@ const resetSubmission = (values, actions) => {
...
+                                    type="text"
+                                    label={localize('Email')}
+                                    background="white"
+                                    placeholder={'example@mail.com'}
+                                    data-lpignore="true"
+                                    required
+                                />
+                            </InputGroup>
+                            <Text align="center" color="red">
+                                {status.error && status.error}
```
Comment (mustofa-binary) [inline src/pages/reset-password/index.js:128]: we can wrap the <Text after && to render text when necessary

File: src/pages/reset-password/index.js [hunk 2]
```diff
@@ -70,82 +78,117 @@ const resetSubmission = (values, actions) => {
...
+                            <InputGroup>
+                                <Input
+                                    id="email"
+                                    name="email"
+                                    error={errors.email}
+                                    value={trimSpaces(values.email)}
+                                    handleError={resetForm}
+                                    onChange={handleChange}
+                                    onBlur={handleBlur}
+                                    autoComplete="off"
```
Comment (mustofa-binary) [inline src/pages/reset-password/index.js:118]: didnt we enable autocomplete for email? 🤔

File: src/pages/reset-password/index.js (hunk 1 above)
Comment (mustofa-binary) [inline src/pages/reset-password/index.js:128]: we can wrap the <Text after && to render text when necessary

File: src/pages/reset-password/index.js [hunk 3]
```diff
@@ -70,82 +78,117 @@ const resetSubmission = (values, actions) => {
...
+                                    placeholder={'example@mail.com'}
+                                    data-lpignore="true"
+                                    required
+                                />
+                            </InputGroup>
+                            <Text align="center" color="red">
+                                {status.error && status.error}
+                            </Text>
+                            <Text align="center" color="green">
+                                {status.success && status.success}
```
Comment (mustofa-binary) [inline src/pages/reset-password/index.js:131]: same here

**Review by mustofa-binary — CHANGES_REQUESTED** [review]

**Review by salar-fs — DISMISSED** [review]

**Review by mustofa-binary — DISMISSED** [review]

File: src/pages/reset-password/index.js [hunk 4]
```diff
@@ -70,82 +83,124 @@ const resetSubmission = (values, actions) => {
...
+                                </StyledButton>
+                            </ButtonContainer>
+                        </Form>
+                    )}
+                </Formik>
+            </StyledContainer>
+        )
+    } else {
+        body = (
+            <FullwidthStyledContainer background="var(--color-pale-gray-64);" mt="7rem">
```
Comment (yashim-deriv) [inline src/pages/reset-password/index.js:165]: Can use --color-grey-30 for this

File: src/themes/variables.js [hunk 5]
```diff
@@ -50,6 +50,7 @@ const Variables = css`
         --color-grey-30: #f5f7fa;
         --color-grey-31: #f8fafc;
         --color-grey-32: #dcdee0;
+        --color-pale-gray-64: rgba(245, 247, 250, 0.64);
```
Comment (yashim-deriv) [inline src/themes/variables.js:53]: the equivalent hex for this is #f5f7fa which is --color-grey-30

**Review by yashim-deriv — CHANGES_REQUESTED** [review]

File: src/themes/variables.js (hunk 5 above)
Comment (reza-fs) [reply src/themes/variables.js:53]: Yes, thanks Yashim, I ended up using `--color-grey-31` which was a match.

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**